### ISSUE TITLE
Optional UIBlurEffectStyle with default value is inconsistent with one constructor

### ISSUE THREAD

**Issue by Daxito** [JanGorman/Agrume#78]
I think UIBlurEffectStyle should have a default value of NIL for all Init Signatures instead of .Dark, the reason is that there is one Init constructor with a default value of NIL but then all of the other Init's have a default value of .Dark and it is not consistent.
Also because the default value for backgroundColor? is also NIL for all the constructors.

Thanks

**Comment by JanGorman** [comment]
yeah, new version moves to an enum with associated value for the background https://github.com/JanGorman/Agrume/blob/4e015235962d0fbe5ca85549e2c871aff1b47ab7/Agrume/Agrume.swift#L8
